modules/ensemble/lib/screen_controller.dart

```diff
@@ -194,6 +194,16 @@ class ScreenController {
         } else if (action.options?['replaceCurrentScreen'] == true) {
           routeOption = RouteOption.replaceCurrentScreen;
         }
+
+        if (action.options?['closeToasts'] ?? true) {
+          ToastController().closeToast();
+        }
+      }
+
+      // No code for NavigateModalScreenAction so need to add this
+      if (action is NavigateModalScreenAction &&
+          (action.options?['closeToasts'] ?? true)) {
```

Comment:
default should be true for both navigateScreen and navigateModalScreen

Reply:
Same as the above comment